.eslintrc

```diff
@@ -1,25 +1,29 @@
-{
-  "parser"  : "babel-eslint",
-  "extends" : [
-    "airbnb"
-  ],
-  "plugins": [
-    "babel",
-    "react"
-  ],
-  "env"     : {
-    "browser" : true
-  },
-  "globals" : {
-    "__DEV__"      : false,
-    "__PROD__"     : false,
-    "__DEBUG__"    : false,
-    "__COVERAGE__" : false,
-    "__BASENAME__" : false
-  },
-  "rules": {
-    "generator-star-spacing": 0,
-    "max-len": [2, 120, 2],
-    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
-  }
-}
```

Comment:
I think that's a good shout. As half the purpose of this app is to help people learn, the more places we can add useful in-place comments, the better!